### PR TITLE
Desktophero room owner cleanup

### PR DESCRIPTION
**Pull request by desktophero**
`identify room owner` function clean-up

### CODE REVIEW

File: spec/lita/handlers/hcadmin_spec.rb [hunk 1]
```diff
@@ -391,46 +409,40 @@
...
-        expect do
-          described_class.id_from_name('floppy drive', room_json)
-        end.to_not raise_error
-      end
-      it 'returns the room_id when given a room name' do
-        expect(described_class.id_from_name('floppy drive', room_json)).to eq 31
-      end
-    end
-  end
+  # describe '#owner_name_from_id' do
```
Comment (pmotch) [inline spec/lita/handlers/hcadmin_spec.rb]: Comment out or remove it?

**Comment by pmotch** [comment]
Red Solo Cup, nice. :+1:

File: spec/lita/handlers/hcadmin_spec.rb (hunk 1 above)
Comment (desktophero) [inline spec/lita/handlers/hcadmin_spec.rb]: Removing that. It's refactored to use "subject" as the object double.

Comment (desktophero) [inline spec/lita/handlers/hcadmin_spec.rb]: Besides ... it's in source control :)

**Comment by desktophero** [comment]
Cool to merge, @pmotch ?

**Comment by pmotch** [comment]
:+1:

**Pull request merged**